helm/thingsboard/templates/node.yaml

```diff
@@ -64,7 +64,7 @@ spec:
         - name: check-db-queue-ready
           image: {{ include "thingsboard.postgresImage" . }}
           command: ['sh', '-c',
-            'export PGPASSWORD=''{{ index .Values "postgresql-ha" "postgresql" "password" }}'' && export row_count=0 && until [ $row_count -ge 3 ]; do sleep 1; row_count=$(psql -h {{ include "thingsboard.pgpoolservicename" . }} -p {{ index .Values "postgresql-ha" "pgpool" "containerPorts" "postgresql" }} -U {{ index .Values "postgresql-ha" "postgresql" "username" }} -d {{ index .Values "postgresql-ha" "postgresql" "database" }} --set=on_error_stop=1 -t -c "SELECT count(*) FROM queue;"); done;']
+            'export PGPASSWORD=''{{ index .Values "postgresql-ha" "postgresql" "password" }}'' && export row_count=0 && until [ $row_count -ge 3 ]; do sleep 1; row_count=$(psql -h {{ include "thingsboard.pgpoolservicename" . }} -p {{ index .Values "postgresql-ha" "pgpool" "containerPorts" "postgresql" }} -U {{ index .Values "postgresql-ha" "postgresql" "username" }} -d {{ index .Values "postgresql-ha" "postgresql" "database" }} --set=on_error_stop=1 -t -c "SELECT count(*) FROM queue WHERE partitions={{ .Values.postgresInitDB.job.check.partitions }};"); done;']
```

Comment:
We need to do the same trick for the MQTT I think.

Reply:
I don't know if it has a init container, but it might need once since it waits directly in the code for the queue table to be created and then caches that info:
![image](https://github.com/midokura/thingsboard-ce-k8s/assets/10707709/1e3f0ddc-2ca3-4922-be0a-c203c26ec889)

Reply:
I saw with Ricard that with the image it's using we can't execute a psql command here, so we'll try to parametrize the partitions